src/common/xo/create-network-modal/index.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import { injectIntl } from 'react-intl'
+import { createSelector } from 'selectors'
+
+import SingleLineRow from '../../single-line-row'
+import _, { messages } from '../../messages'
+import { SelectPif } from '../../select-objects'
+import { Col } from '../../grid'
+
+class CreateNetworkModalBody extends Component {
+  _getPifPredicate = createSelector(
+    () => {
+      const container = this.props.container
+      return container.type === 'pool' ? container.master : container.id
+    },
+    hostId => pif => pif.$host === hostId && pif.vlan === -1
+  )
+
+  get value () {
+    const { refs } = this
+    const { container } = this.props
+    return {
+      pool: container === 'pool' ? container.id : container.$pool,
+      name: refs.name.value,
+      description: refs.description.value,
+      pif: refs.pif.value.id,
+      mtu: refs.mtu.value,
+      vlan: refs.vlan.value
+    }
+  }
+
+  render () {
+    const { formatMessage } = this.props.intl
+    return <div>
+      <SingleLineRow>
+        <Col size={6}>{_('newNetworkInterface')}</Col>
+        <Col size={6}>
+          <SelectPif
+            predicate={this._getPifPredicate()}
+            ref='pif'
+          />
+        </Col>
+      </SingleLineRow>
+      &nbsp;
```

Comment:
I can't.
Apparently, `{' '}` is not removed only if it is placed between inline blocks. In my case, I need `&nbsp`.